[Docs] Cheatsheet - Ingress should use network().v1() api calls (defaults to v1beta1)

### Describe the bug

Currently in 6.3.1 version my IDE was failing to get ingress resources with api v1, as it was using v1beta1 by default (my assumption as far from k-client expert).
Changing code from Cheatsheet example to `network().v1().ingress()` worked as expected. 

### Fabric8 Kubernetes Client version

6.3.1

### Steps to reproduce

1.Will expect v1beta1 api: ` ingresses = client.network().ingresses().inNamespace(namespaceName).list()`

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.1:compile (default-compile) on project brokerqe: Compilation failure: Compilation failure: 
[ERROR] /home/mtoth/work/repos/claire/src/main/java/io/brokerqe/KubeClient.java:[367,102] incompatible types: io.fabric8.kubernetes.api.model.networking.v1beta1.Ingress cannot be converted to io.fabric8.kubernetes.api.model.networking.v1.Ingress
[ERROR] /home/mtoth/work/repos/claire/src/main/java/io/brokerqe/KubeClient.java:[379,102] incompatible types: inference variable T has incompatible bounds
[ERROR]     equality constraints: io.fabric8.kubernetes.api.model.networking.v1.Ingress
[ERROR]     lower bounds: io.fabric8.kubernetes.api.model.networking.v1beta1.Ingress
```

### Expected behavior

Adding `v1()` - solves the issue: `ingresses = client.network().v1().ingresses().inNamespace(namespaceName).list()`

### Runtime

minikube

### Kubernetes API Server version

1.23

### Environment

Linux

### Fabric8 Kubernetes Client Logs

```
.fabric8.kubernetes.client.KubernetesClientException: Failure executing: GET at: https://192.168.49.2:8443/apis/networking.k8s.io/v1beta1/namespaces/tls-tests/ingresses. 
Message: the server could not find the requested resource. 
Received status: Status(apiVersion=v1, code=404, details=StatusDetails(causes=[], group=null, kind=null, name=null, retryAfterSeconds=null, uid=null, additionalProperties={}), kind=Status, message=the server could not find the requested resource, metadata=ListMeta(_continue=null, remainingItemCount=null, resourceVersion=null, selfLink=null, additionalProperties={}), reason=NotFound, status=Failure, additionalProperties={}).
```


### Additional context

_No response_

--- a/doc/CHEATSHEET.md
+++ b/doc/CHEATSHEET.md
@@ -910,14 +910,14 @@ client.serviceAccounts().inNamespace("default").withName("serviceaccount1").dele
 ```
 
 ### Ingress
-`Ingress` resource is available in Kubernetes Client API via `client.network().ingress()`. Here are some examples regarding its usage:
+`Ingress` resource is available in Kubernetes Client API via `client.network().v1().ingress()`. Here are some examples regarding its usage:
 - Load `Ingress` from yaml:
 ```
-Ingress ingress = client.network().ingress().load(new FileInputStream("ingress.yml")).get();
+Ingress ingress = client.network().v1().ingress().load(new FileInputStream("ingress.yml")).get();
 ```
 - Get `Ingress` from Kubernetes API server:
 ```
-Ingress ingress = client.network().ingress().inNamespace("default").withName("ingress1").get();
+Ingress ingress = client.network().v1().ingress().inNamespace("default").withName("ingress1").get();
 ```
 - Create `Ingress`:
 ```
@@ -933,27 +933,27 @@ Ingress ingress = new IngressBuilder()
   .endRule()
   .endSpec()
   .build();
-client.network().ingress().inNamespace("default").create(ingress);
+client.network().v1().ingress().inNamespace("default").create(ingress);
 ```
 - Create or Replace `Ingress`:
 ```
-Ingress igx = client.network().ingress().inNamespace("default").createOrReplace(ingress);
+Ingress igx = client.network().v1().ingress().inNamespace("default").createOrReplace(ingress);
 ```
 - List `Ingress` in some namespace:
 ```
-IngressList ingressList = client.network().ingress().inNamespace("default").list();
+IngressList ingressList = client.network().v1().ingress().inNamespace("default").list();
 ```
 - List `Ingress` in any namespace:
 ```
-IngressList ingressList = client.network().ingress().inAnyNamespace().list();
+IngressList ingressList = client.network().v1().ingress().inAnyNamespace().list();
 ```
 - List `Ingress` with some label in any namespace:
 ```
-IngressList ingressList = client.network().ingress().inNamespace("default").withLabel("foo", "bar").list();
+IngressList ingressList = client.network().v1().ingress().inNamespace("default").withLabel("foo", "bar").list();
 ```
 - Delete `Ingress`:
 ```
-client.network().ingress().inNamespace("default").withName("ingress1").delete();
+client.network().v1().ingress().inNamespace("default").withName("ingress1").delete();
 ```
 
 ### StatefulSet

